scripts/ship_it.sh

```
@@ -0,0 +1,195 @@
+    echo "⏭️ Branch updated successfully. Skipping Slack notification (team will see updates in GitHub automatically)."
+else
+    echo "🌿 Opening an official Pull Request on $REPO_ORG/$REPO_NAME via GitHub API..."
+    
+    PR_URL=$(gh pr create \
+        --repo "$REPO_ORG/$REPO_NAME" \
+        --head "$MY_GITHUB_USER:$BRANCH_NAME" \
+        --base "main" \
+        --title "$COMMIT_MSG" \
+        --body "Automated PR created by crane pipeline." 2>/dev/null || echo "")
+
+    if [ -n "$PR_URL" ]; then
```

Shellcheck (0.11.0)

[info] The mentioned syntax error was in this then clause.

(SC1009)